src/redux/slices/transactions/index.js

```diff
@@ -1,7 +1,103 @@
+import { createAsyncThunk, createSlice } from '@reduxjs/toolkit';
+import set from 'lodash-es/set';
 import { createSelector } from 'reselect';
 
+import { getTransactions, transactionExtraInfo } from '../../../utils/explorer-api';
+
 const SLICE_NAME = 'transactions';
 
+const initialState = {
+    transactions: {
+        byAccountId: {}
+    },
+    status: {
+        
+    }
+};
+
+const fetchTransactions = createAsyncThunk(
```

Comment:
Feels now that we have loading state in Redux, we should take advantage of that to add a `condition` option to this like we did to the NFT slice, so that we don't dispatch concurrent requests to load the same transactions.

However, to be sure that we don't fail to fetch transactions for an account when we switch to it while there is a pending attempt to load transactions for the account that we were previously viewing, we would need a `status` object to exist _on each entry under `byAccountId`_, rather than as a root construct... which would mean our state shape would look like:
```
{
  transactions: {
    byAccountId: {
      accountName1: {
        items: [...transactionEntries],
        status: { loading, error }
      },
      account2Name: {
        items: [...transactionEntries],
        status: { loading, error }
      },
      ...etc
    }
  }
}

```

This shape also gives us the ability to know which account we failed to load for since we can also track `error` state per account slice; it's basically the same pattern as the nftSlice uses now.

Reply:
Done.